### PR TITLE
Add a description on the API calls to list / delete the API keys 

### PR DESCRIPTION
**Pull request by yomovh**

### CODE REVIEW

File: pages/manage_and_operate/api/first-steps/guide.en-gb.md [hunk 1]
```diff
@@ -195,6 +195,13 @@ client = ovh.Client(
 print("Welcome", client.get('/me')['firstname'])
 ```
 
+#### List and revoke your keys
+
+There is currently no Control Panel to list and revoke your keys. The API portal can be used to do that : 
+* List the keys ids through [GET /me/api/application](https://eu.api.ovh.com/console-preview/?section=%2Fme&branch=v1#get-/me/api/application)
```
Comment (amstuta) [inline pages/manage_and_operate/api/first-steps/guide.en-gb.md]: you could use formatted links to the console, like this: `@api {v1} /me GET /me/api/application/{applicationId}`, what do you think ?

**Review by amstuta — COMMENTED** [review]
Otherwise, ok for me

**Review by gbarideau — APPROVED** [review]

**Review by bs-ovh — APPROVED** [review]

File: pages/manage_and_operate/api/first-steps/guide.de-de.md [hunk 2]
```diff
@@ -199,6 +199,31 @@ client = ovh.Client(
...
 
+#### Schlüssel auflisten und widerrufen
+
+Es gibt derzeit keine Option im Kundencenter, mit der Sie Ihre Schlüssel auflisten und widerrufen können. Dazu kann das API-Portal verwendet werden:
+
+- Erhalten Sie die Schlüssel-IDs mit dem folgenden Aufruf:
+
+> [!api]
+>
+> @api {v1} /me GET /me/api/application/
```
Comment (amstuta) [inline pages/manage_and_operate/api/first-steps/guide.de-de.md]:
```suggestion
> @api {v1} /me GET /me/api/application
```

I don't think the resulting URL will work with this trailing slash

Comment (OVH-Team-Guides) [reply pages/manage_and_operate/api/first-steps/guide.de-de.md]: Thank you @amstuta , I fixed this

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @yomovh

**Pull request merged**